hw02_unpack_string/go.mod

```
@@ -1,4 +1,4 @@
-module github.com/fixme_my_friend/hw02_unpack_string
+module github.com/adettelle/hw02_unpack_string
```

Ещё нужно указать название репозитория, чтобы получить путь, по которому модуль можно найти
```suggestion
module github.com/adettelle/hw/hw02_unpack_string
```

---

Ок